_events/sr2024/first-challenge-submission-deadline.md

```diff
@@ -0,0 +1,17 @@
+---
+title: First Challenge Submission Deadline
+date: 2023-12-09 18:00:00 Europe/London
+layout: event
+type: challenge-submission
+location: Discord
```

Comment:
Do we want to specifically call out the challenge-submissions channel? I realise it might be a bit verbose though; this is definitely ok as-is.

Reply:
I think that may be a little verbose